modules/http/src/main/java/org/mule/module/http/internal/listener/DefaultHttpListenerConfig.java

```diff
@@ -338,4 +338,15 @@ public void setConnectionIdleTimeout(int connectionIdleTimeout)
     {
         this.connectionIdleTimeout = connectionIdleTimeout;
     }
+    
+    @Override
+    public String toString() {
+        StringBuilder buf = new StringBuilder();
+        buf.append("org.mule.module.http.internal.listener.DefaultHttpListenerConfig{name=")
```

Comment:
I don't think the full class name is necessary.

Reply:
I sent a change to remove the package and leave only the classname.